README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# hw01
+## General Bio
+Hello, my name is Jackson (though I go by Jay) Gibbs. I am a fourth year quadruple major studying:
+* East Asian Languages and Civilizations 
+* Public Policy
+* Political Science
+* Comparitive Race and Ethnic Studies
+
+### Why I am taking the course
```

Comment:
Very good use of the syntax for headers. I used another format in mine and turns out it's much simpler if we just use `###`